docs/advanced-features/middleware.md

```diff
@@ -1,102 +1,361 @@
 ---
-description: Learn how to use Middleware in Next.js to run code before a request is completed.
+description: Learn how to use Next.js Middleware to run code before a request is completed.
 ---
 
-# Middleware (Beta)
+# Next.js Middleware
 
 <details open>
-  <summary><b>Version History</b></summary>
 
-| Version   | Changes                                                                                    |
+<summary><b>Version History</b></summary>
+
+| Version | Changes |
+
 | --------- | ------------------------------------------------------------------------------------------ |
+
+| `v12.2.0` | Next.js Middleware GA. |
```

Comment:
Version number of GA is subject to change.

Reply:
Cool, noted!